internal/node/node.bzl

```diff
@@ -390,69 +390,39 @@ remote debugger.
 # See https://github.com/bazelbuild/buildtools/issues/471#issuecomment-485283200
 
 def nodejs_binary_macro(name, data = [], args = [], visibility = None, tags = [], testonly = 0, **kwargs):
-    """This macro exists only to wrap the nodejs_binary as an .exe for Windows.
+    """This macro exists only to wrap the nodejs_binary to pull in the runfiles helper
 
     This is exposed in the public API at `//:defs.bzl` as `nodejs_binary`, so most
     users loading `nodejs_binary` are actually executing this macro.
 
     Args:
       name: name of the label
       data: runtime dependencies
-      args: applied to the wrapper binary
-      visibility: applied to the wrapper binary
-      tags: applied to the wrapper binary
-      testonly: applied to nodejs_binary and wrapper binary
       **kwargs: passed to the nodejs_binary
     """
-    all_data = data + ["@bazel_tools//tools/bash/runfiles"]
-
     nodejs_binary(
-        name = "%s_bin" % name,
-        data = all_data,
+        name = name,
+        data = data + ["@bazel_tools//tools/bash/runfiles"],
```

Comment:
Any reason we could not add this as an implicit dependency to the rules to completely get rid of the macro?

Reply:
yes certainly, I was being lazy